src/webapp/components/dataset-wizard/IndicatorsDataSet.tsx

```diff
@@ -220,7 +217,13 @@ export const IndicatorsDataSet = React.memo((props: IndicatorsDataSetProps) => {
                             onClose={() => setShowFilterModal(false)}
                             indicatorsPerCompetency={indicatorsPerCompetency}
                             indicatorsPerType={indicatorsPerType}
-                            hide={companionTable}
+                            hidden={companionTable}
```

Comment:
It looks weird. companionTable is a boolean, do we have a better name for the prop? `hasCompanionTable` or something like this? you know more the context